roles/taca/templates/site_fake_delivery.yml.j2

```diff
@@ -0,0 +1,85 @@
+log:
+    file: "{{ ngi_pipeline_site_path }}/log/taca.log"
+
+deliver:
+    rootpath: "/proj/{{ ngi_pipeline_site_delivery }}/nobackup/NGI"
+    analysispath: <ROOTPATH>/ANALYSIS/<PROJECTID>
+    datapath: <ROOTPATH>/DATA/<PROJECTID>
+    stagingpath: <ROOTPATH>/DELIVERY/<PROJECTID>
+    deliverypath: /proj/<UPPNEXID>/INBOX/<PROJECTID>
```

Comment:
On Irma4, this delivery path should currently be `/pica/sw/links/proj/<UPPNEXID>/INBOX/<PROJECTID>`